src/pages/PoolDetails/PoolDetailsStats.tsx

```diff
@@ -0,0 +1,229 @@
+import { Trans } from '@lingui/macro'
+import Column from 'components/Column'
+import CurrencyLogo from 'components/Logo/CurrencyLogo'
+import Row from 'components/Row'
+import { DeltaArrow, formatDelta } from 'components/Tokens/TokenDetails/Delta'
+import { PoolData } from 'graphql/thegraph/PoolData'
+import { useCurrency } from 'hooks/Tokens'
+import { useColor } from 'hooks/useColor'
+import { useScreenSize } from 'hooks/useScreenSize'
+import { ReactNode, useMemo } from 'react'
+import { Text } from 'rebass'
+import styled, { css } from 'styled-components'
+import { BREAKPOINTS } from 'theme'
+import { ThemedText } from 'theme/components'
+import { NumberType, useFormatter } from 'utils/formatNumbers'
+
+const HeaderText = styled(Text)`
```

Comment:
should we leverage `ThemedText` here instead of raw rebass https://github.com/Uniswap/interface/blob/main/src/theme/components/text.tsx#L17

Reply:
We currently don't have a ThemedText that matches this criteria. I could add it for this currently one-off?

Reply:
ah okay probably fine as is

Reply:
@infredible thoughts on one-off? From a maintenance perspective it would be great if this aligned w/ our text theme.

Reply:
We shouldn't be using any one-offs on this page. This should be heading3

Reply:
@infredible I think heading3 might be out of date since the spore work as in the Figma its showing as fontWeight 500. Do you know if it should match up to one of [these Header styles](https://github.com/Uniswap/interface/blob/main/src/theme/components/text.tsx#L17) we currently have?
![Screenshot 2023-09-27 at 11 24 19 AM](https://github.com/Uniswap/interface/assets/11512321/09b3e637-047f-4fc5-83a3-411e7ba28de8)